tokens/src/components/Button/core.json

```diff
@@ -72,6 +72,6 @@
     }
   },
   "transition": {
-    "btn": { "value": "null", "source": "$btn-transition" }
+    "btn": { "value": "0", "source": "$btn-transition" }
```

Comment:
I think we should use `none` here instead

`0` is not a valid value
![image](https://user-images.githubusercontent.com/52399399/209926549-bfd71f72-8c56-4eb9-a01d-f83366f9a32d.png)

Reply:
Corrected, thanks